database/factories/LocationFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Http\Models\Location;
+use Faker\Generator as Faker;
+
+$factory->define(Location::class, function (Faker $faker) {
+    return [
+        'apartment_number' => random_int(1,100),
```

Comment:
```suggestion
        'apartment_number' => random_int(1, 100),
```
sửa hết những chỗ tương tự